### ISSUE TITLE
Support loading config from a separate config file

### ISSUE THREAD

**Issue by ianjsikes** [dj95/zjstatus#86]
**Is your feature request related to a problem? Please describe.**
My config for the status bar is huge, and I don't want to have to scroll past a bunch of ugly color-formatted strings and special box-drawing characters every time I need to edit my Zellij layouts.

**Describe the solution you'd like**
I want to be able to specify my status bar config in a separate file. When I load the status bar in my layouts, it will read from that file first, then override it with any config options that are specified in the layout itself.

**Additional context**
It seems like Zellij plugins support [reading from the filesystem](https://zellij.dev/documentation/plugin-api-file-system).

**Comment by dj95** [comment]
Hi and thanks for your feature request.

I got a similar one in the past, where I described, why that is unfortunately not possible to implement zjstatus.

https://github.com/dj95/zjstatus/issues/75

**TL;DR** Zellij creates a sandbox for each plugin and exposes just a few paths to it. Therefore the config would need to exist in the folder, in which zellij is started. As this will vary, you cannot reliably implement an external config file.
